backend/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:slim
+
+WORKDIR /app
+
+COPY package.json .
+
+COPY package-lock.json .
+
+RUN npm install && npm install -g nodemon
```

Comment:
Running with nodemon is not recommended for production because it consumes a LOT of memory, and the orchestration tool (or even Heroku) should do the job to restart the process in case it fail

Reply:
nodemon is good for development, could also use `forever` or `pm2` ... but I think we will probably move to AWS EKS or EC2 instead of Heroku